packages/core/modules-sdk/src/loaders/utils/load-internal.ts

```diff
@@ -208,17 +208,24 @@ export async function loadModuleMigrations(
 async function importAllFromDir(path: string) {
   let filesToLoad: string[] = []
 
+  const excludedExtensionsRegexp = /(\.ts\.map|\.js\.map|\.d\.ts)$/
+
   await readdir(path).then((files) => {
     files.forEach((file) => {
-      if (file !== "index.js" && file.endsWith(".js")) {
-        const filePath = join(path, file)
-        const stats = statSync(filePath)
-
-        if (stats.isDirectory()) {
-          // TODO: should we handle that? dont think so but I put that here for discussion
-        } else if (stats.isFile()) {
-          filesToLoad.push(filePath)
-        }
+      if (
+        file.startsWith("index.") ||
```

Comment:
We could also do `index.js || index.ts`, since that's what we are doing below as well, no need for Regex IMO

Reply:
we dont want the index.ts.map nor the index.js.map etc and we dont want any files not index but ending with those extensions

Reply:
for example we don't want to import my-service.ts.map

Reply:
@adrien2p ah sorry I wasn't clear, I meant we can say `file === "index.js" || file === "index.ts"`, since later down the code we replace the index.js and index.ts explicitly. Anyway it's not a big deal, I just thought maybe we can match the two

Reply:
we are replacing it only for the module path, here it is the resources :)